static/js/protein/chart.ts

```diff
@@ -29,10 +31,63 @@ export function drawTissueScoreChart(
   id: string,
   data: { name: string; value: string }[]
 ): void {
-  // TODO: convert data to tmpData format.
-  const tmpData = [{ x1: 20, x2: 60, y1: 30, y2: 50 }];
+  const testData = data;
+  /*
+  Convert the tissue expression level to numerical values 
+  * @param val value of the data array 
+  */
+  function convertData(val) {
+    if (val == "ProteinExpressionNotDetected") {
+      return 0;
+    } else if (val == "ProteinExpressionLow") {
+      return 1;
+    } else if (val == "ProteinExpressionMedium") {
+      return 2;
+    } else if (val == "ProteinExpressionHigh") {
+      return 3;
+    }
+  }
+
+  interface newData{
```

Comment:
- this interface should be defined outside of this function and exported because it can be used in mulitple places across multiple files. eg. line 32 in this file,  line 73 and 91 in page.tsx

- this name is very ambiguous. When defining interfaces, we want to use a name that gives other develops an idea of what the kind of object could be. Here maybe something along the lines of ProteinChartDataPoint
    - also when naming interfaces, we want to use pascal case

Reply:
Added two interfaces because the one used in the chart.ts returns a number value while the one used in page.tsx returns string values.